server/api/index.js

```diff
@@ -0,0 +1,61 @@
+const express = require("express");
+const mongoose = require("mongoose");
+const Recipe = require("./models/recipe");
+require('dotenv').config();
```

Comment:
https://zeit.co/docs/v2/serverless-functions/env-and-secrets this is how now suggests doing env variables, not sure which one we want to go with

Reply:
Definitely secrets

Reply:
Switched from .env to now secrets